nncf/quantization/algorithms/min_max/utils.py

```diff
@@ -68,9 +104,10 @@ def calculate_weight_quantizer_parameters(weight_tensor: np.ndarray, quantizer_c
         axes = None
     input_high = np.amax(weight_tensor, axis=axes)
     input_low = np.amin(weight_tensor, axis=axes)
-    scales = calculate_scale_level(input_high, input_low, num_bits, mode)
-    zero_points = np.zeros_like(scales, dtype=np.int64)
-    return QuantizerLayerParameters(scales.tolist(), zero_points.tolist(), mode, axis)
+    tensor_type = np.uint8 if np.all(input_low >= 0) else np.int8
```

Comment:
OpenVINO supports np.uint8 on the weights?

Reply:
Make it strictly int8